src/test/java/com/tikal/hudson/plugins/notification/ProtocolTest.java

```diff
@@ -170,9 +171,10 @@ private UrlFactory startSecureServer(Servlet servlet, String path, String author
     ServerConnector connector = new ServerConnector(server);
     server.setConnectors(new Connector[] {connector});
 
-    ServletHandler servletHandler = new ServletHandler();
-    server.setHandler(servletHandler);
-    servletHandler.addServletWithMapping(new ServletHolder(servlet), path);
+    ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
+    context.setContextPath("/");
+    context.addServlet(new ServletHolder(servlet), path);
+    server.setHandler(context);
```

Comment:
I'm using `ServletContextHandler` because `server.setHandler(servletHandler);` isn't working anymore after I've changed the import to `jetty.ee8.servlet` (`Cannot resolve method 'setHandler(ServletHandler)'`) it seems that it doesn’t match the handler type it expects with EE8.